terra-jupyter-aou/Dockerfile

```diff
@@ -266,6 +266,16 @@ RUN PERL_MM_USE_DEFAULT=1 cpan Text::Soundex
 RUN curl -L http://www.repeatmasker.org/RepeatMasker/RepeatMasker-4.1.4.tar.gz \
   | (cd /usr/local; tar -xz)
 
+
+# Install Cromshell
+RUN cromshell-alpha version
```

Comment:
this doesn't work, I got this error locally

Step 44/49 : RUN cromshell-alpha version
 ---> Running in d3cf1b1fa50d
/bin/sh: 1: cromshell-alpha: not found
The command '/bin/sh -c cromshell-alpha version' returned a non-zero code: 127

Reply:
right, we decide to use Cromshell-beta: https://pmi-engteam.slack.com/archives/CHRN2R51N/p1678994326910319
I will revert this change